launch/upload_description.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<launch>
+  <arg name="__ns" default=""/>
+  <arg name="prefix" default="$(arg __ns)_"/>
+
+  <param if="$(eval prefix == '_')" name="robot_description" command="$(find xacro)/xacro $(find robotont_description)/urdf/robotont_base.urdf.xacro"/>
+  <param unless="$(eval prefix == '_')" name="robot_description" command="$(find xacro)/xacro $(find robotont_description)/urdf/robotont_base.urdf.xacro prefix:=$(arg prefix)"/>
+</launch>
```

Comment:
github toob välja, et faili lõpus pole reavahetust, kas see on mingis olukorras probleem?

Reply:
> A text file, under unix, consists of a series of lines, each of which ends with a newline character (\n). A file that is not empty and does not end with a newline is therefore not a text file.

Siit postitusest: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
@mmaao seega oleks targem alati viimase tekstisisaldava rea lõppu üks enter panna